torch/nn/quantized/modules/conv.py

```diff
@@ -0,0 +1,93 @@
+# coding=utf-8
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+from torch._jit_internal import weak_module, weak_script_method
+from torch._ops import ops
+from torch.nn.modules.module import Module
+from torch.nn.modules.utils import _pair
+from torch.nn.quantized import functional as qF
+
+"""Computes the output shape given convolution parameters."""
+def _conv_output_shape(input_size, kernel_size, padding, stride, dilation,
+                       output_padding=0):
+    return np.floor((input_size + 2 * padding - kernel_size - (kernel_size - 1)
+                    * (dilation - 1)) / stride) + 2 * output_padding + 1
+
+@weak_module
+class _ConvNd(Module):
+    def __init__(self, weight, bias, scale, zero_point, dtype,
+                 stride, padding, dilation, transposed, output_padding,
+                 groups, padding_mode):
+        if transposed:
+            raise NotImplementedError("Transposed convolution not implemented!")
+        super(_ConvNd, self).__init__()
+        # self.__name__ = "qConvolution"
+        self.in_channels = weight.shape[1] * groups
+        self.out_channels = weight.shape[0]
+        self.kernel_size = weight.shape[2:]
+        self.stride = stride
+        self.padding = padding
+        self.dilation = dilation
+        self.transposed = transposed
+        self.output_padding = output_padding
+        self.groups = groups
+        self.padding_mode = padding_mode
+
+        weight = ops.quantized.fbgemm_conv_prepack(weight, groups)
```

Comment:
Please use _packed as a prefix for packed tensors. 
self._packed_weight = weight